components/tooltip/tooltip-help.js

```diff
@@ -0,0 +1,87 @@
+import '../colors/colors.js';
+import '../tooltip/tooltip.js';
+import { css, html, LitElement } from 'lit';
+import { FocusMixin } from '../../mixins/focus-mixin.js';
+import { FocusVisiblePolyfillMixin } from '../../mixins/focus-visible-polyfill-mixin.js';
+
+/**
+ * A component used to display additional information when users focus or hover over some text.
+ * @slot - Default content placed inside of the tooltip
+ */
+class HelpTooltip extends FocusMixin(FocusVisiblePolyfillMixin(LitElement)) {
+
+	static get properties() {
+		return {
+			/**
+			 * REQUIRED: Text that will render as the Help Tooltip opener
+			 * @type {string}
+			 */
+			text: { type: String }
+		};
+	}
+
+	static get styles() {
+		return [css`
+			:host {
+				color: var(--d2l-color-tungsten);
+				display: inline-block;
+				font-size: 0.7rem;
+				font-weight: 400;
+				line-height: 1rem;
+				margin: auto;
+				text-decoration-color: var(--d2l-color-galena);
+			}
+			:host([hidden]) {
+				display: none;
+			}
+			#d2l-tooltip-help-text {
+				background: none;
+				border: none;
+				color: inherit;
+				font-family: inherit;
+				font-size: inherit;
+				font-weight: inherit;
+				line-height: inherit;
+				padding: 0;
+				text-decoration-color: inherit;
```

Comment:
super nit: based on https://www.w3schools.com/cssref/pr_text_text-decoration.asp it looks like these `text-decoration` properties can all be combined into just `text-decoration` rather than be split over multiple properties.

Reply:
IIRC, I think I discovered that for some strange reason, Safari didn't like the short-hand version. I found that it worked by specifying the properties individually.  I don't mind double checking that though.

Reply:
Ah interesting - thanks for looking into that

Reply:
Yep initially we used the shorthand but after Dave testing the component out on safari, we that that property was just ignored and resorted to styling it all out separately as a work around.